sdk/python/feast/feature_store.py

```diff
@@ -1406,7 +1407,8 @@ def push(
     def write_to_online_store(
         self,
         feature_view_name: str,
-        df: pd.DataFrame,
+        df: Optional[pd.DataFrame] = None,
+        input_dict: Optional[Dict] = None,
```

Comment:
Wouldn't this make more sense as a single param with a Union type?

Reply:
We can do a staggered release for this and get feedback from folks but I wouldn't want to make any breaking changes.

Reply:
yeah, `df` name is really awkward. maybe call the new one something other than `input_dict` so that it can become the universal one after we deprecate and remove `df`? just `inputs` for example.

Reply:
Yeah I like that. I'll update.